addons/GearSwap/libs/closetCleaner.lua

```diff
@@ -0,0 +1,367 @@
+--Copyright (c) 2016-2017, Brimstone
```

Comment:
Please adjust `(c)` to `©`. Are you `Brimstone`, or is that a copy/paste oversight?